target/linux/airoha/config-5.15

```diff
@@ -173,6 +169,8 @@ CONFIG_MTD_UBI_BLOCK=y
 CONFIG_MTD_UBI_WL_THRESHOLD=4096
 CONFIG_MUTEX_SPIN_ON_OWNER=y
 CONFIG_NEED_DMA_MAP_STATE=y
+CONFIG_NET_FLOW_LIMIT=y
+CONFIG_NET_SELFTESTS=y
```

Comment:
smell like generic config?

Reply:
CONFIG_NET_FLOW_LIMIT is enabled by CONFIG_SMP.
CONFIG_NET_SELFTESTS is enabled by CONFIG_NET_DSA.
Many targets support SMP or DSA therefore these symbols are enabled there.
There are subtargets without DSA and SMP e.g. at91, lantiq/ase, lantiq/xway, lantiq/xway_legacy, realtek/rtl838x, uml....

Reply:
Sorry for the delay... wonder if selftests is needed?